### ISSUE TITLE
[Refresh view - part 1] Changing tag color in the sidebar, should automatically update the respective folders and files color tags in the view and vice versa

### ISSUE THREAD

**Issue by 4jNsY6fCVqZv** [elementary/files#1012]
I'm just wondering if both should always be possible? Just as renaming is also possible in both views.

**Comment by jeremypw** [comment]
> Just as renaming is also possible in both views.

Renaming in the sidebar does something different to renaming in the folder view.  The former only changes the name of the bookmark (for example you might want an abbreviated form of the target).  The latter changes the name of the folder or file and (at the moment) renders any associated bookmark invalid.

**Comment by jeremypw** [comment]
It would be possible to change the tag color both in the sidebar and in the view though.

**Comment by 4jNsY6fCVqZv** [comment]
So we can close this issue when https://github.com/elementary/files/pull/1010 is resolved, right?

**Comment by jeremypw** [comment]
Yes, I would expect to synchronize sidebar and view via the plugin in that PR once the appearance is agreed.

**Comment by 4jNsY6fCVqZv** [comment]
super nice!

**Comment by jeremypw** [comment]
Closing as having tag colors in the sidebar is not yet a thing.  This issue can be reopened if sidebar color tags are implemented.
